src/services/auth.ts

```diff
@@ -41,6 +40,20 @@ export const auth = {
 
         return response;
     },
+    getAuthHeader() {
+        const authDetails = auth.getAuthDetails();
+        let response;
+
+        if (authDetails?.session) {
+            const token = window.btoa(
+                `${authDetails.session.account_id}:${authDetails.session.token}`
+            );
+
+            response = `Basic ${token}`;
+        }
+
+        return response;
+    },
```

Comment:
Axios no longer handles Auth headers for us so we need to generate them now. Decided to put this in the auth service just because it seems heavily related even though it will only ever be used by the client.